.github/workflows/ci.yml

```diff
@@ -73,8 +72,7 @@ jobs:
     - name: cargo nextest run
       run: |
         cargo nextest run \
-          --workspace \
-          --all-features
```

Comment:
does removing this mean that the forth3 tests aren't run with async, floats, etc enabled? because if that's the case, i would prefer an approach that lets us keep the `--all-features` build.

Reply:
The miri tests do still set this, but the normal tests don't. I'll rework this PR and add a mnemos-tracing crate instead.